src/ng/sce.js

```diff
@@ -288,17 +322,23 @@ function $SceDelegateProvider() {
      * @name $sceDelegate#trustAs
      *
      * @description
-     * Returns an object that is trusted by AngularJS for use in specified strict
-     * contextual escaping contexts (such as ng-bind-html, ng-include, any src
-     * attribute interpolation, any dom event binding attribute interpolation
-     * such as for onclick,  etc.) that uses the provided value.
-     * See {@link ng.$sce $sce} for enabling strict contextual escaping.
+     * Marks the parameter as trusted for the specified context, and returns the
+     * trusted representation of it. This trusted object should later on be
+     * used as-is, without any security check, by sinks that use this context.
```

Comment:
I've replaced it by "used as is without any security checks by bindings or directives that require this security context."